vllm/model_executor/layers/fused_moe/layer.py

```diff
@@ -315,6 +332,129 @@ def forward_tpu(
     forward_native = forward_tpu if current_platform.is_tpu() else forward_cuda
 
 
+@CustomOp.register("unquantized_fused_cutlass_moe")
+class UnquantizedFusedCutlassMoEMethod(FusedMoEMethodBase, CustomOp):
+    """CUTLASS MoE method without quantization."""
+
+    @staticmethod
+    def check_supported(activation: str,
+                        expert_map: Optional[torch.Tensor],
+                        error: bool = True) -> bool:
+        required_capability = 90
+        capability_tuple = current_platform.get_device_capability()
+
+        if capability_tuple is not None:
+            capability = capability_tuple.to_int()
+            arch_supported = (capability == required_capability
```

Comment:
does this need to be a straight equality check?  could it be `>=`?

Reply:
These CUTLASS kernels aren't forward-compatible unfortunately